src/mobility/queries/use-geofencing-zones.tsx

```diff
@@ -0,0 +1,20 @@
+import {useQuery} from '@tanstack/react-query';
+import {getGeofencingZones} from '@atb/api/mobility';
+import {useGeofencingZonesEnabled} from '../use-geofencing-zones-enabled';
+
+const ONE_MINUTE = 1000 * 60;
```

Comment:
Could we cache this longer? I'm guessing geofencing zones doesn't change that often?

Reply:
Yes, how long would you suggest?

Reply:
12 hours?